docker/build/bin/km-build

```diff
@@ -8,14 +12,14 @@ set -vx
 GOPKG=github.com/mesosphere/kubernetes-mesos
 
 test ${#} -eq 0 && CMD=( make bootstrap install ) || CMD=( "${@}" )
-test -n "$GOPATH" || GOPATH=/pkg
+test -n "$GOPATH" || GOPATH=/go
```

Comment:
I've never seen `/pkg` used as a GOPATH before. Usually Go apps don't need to know where the gopath is. So the only things aware are build scripts. I'm pretty sure km-build (renamed from hack/dockerbuild/run.sh) was the only thing using `/pkg`, because that's what the Dockerfile was setting. So it was the only thing I needed to change. 

That said, `km-build` probably shouldn't have a default in it at all. I'll change it to explode if not set. 

FYI: Go generally requires GOPATH to be set (at compilation time, not runtime), but the conventional defaults are `/go` (single-user envs, like docker images) or `~/go` (multi-user envs, like a workstation).